src/state/store.ts

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+// import puppylogReducer from './Puppylog/puppylogSlice';
+
+export default configureStore({
+  reducer: {
+    // puppylog: puppylogReducer,
```

Comment:
uncomment these lines when correct the above errors

Reply:
fixed